clientSide/theteddyproject/src/App.js

```diff
@@ -1,11 +1,11 @@
 
 import './App.css';
+import Route from './Components/Routes/Routes';
 
 function App() {
   return (
     <div className="App">
-      <h1>Hai I am FOuziya</h1>
```

Comment:
please remove unwanted lines before pushing

Reply:
Okay i will remove the unwanted lines